### PR TITLE
Softlayer bosh doc enhancement

### PR DESCRIPTION
**Pull request by gu-bin**

### CODE REVIEW

**Comment by gu-bin** [comment]
@cppforlife Would you please merge this PR? Thanks.

/cc @mattcui @dr.max

File: init-softlayer.html.md.erb [hunk 1]
```diff
@@ -190,16 +190,18 @@ VLANs provide the ability to partition devices and subnets on the network. To or
 
 1. Install [bosh-init](./install-bosh-init.html).
 
+1. Establish VPN connection between your host and Softlayer. The machine where to run bosh-init needs to communicate with the target director VM over the SoftLayer private network.
```
Comment (maximilien) [inline init-softlayer.html.md.erb]: @gu-bin  anyway to add a link from SL docs or blog post on how to do this?

File: softlayer-cpi.html.md.erb [hunk 2]
```diff
@@ -169,5 +185,7 @@ compilation:
 
 The ``Bosh_ip`` property specified under ``cloud_properties`` is used for Softlayer CPI to differentiate the director and common vms. The one with cloud_property ``Bosh_ip`` is a common vm. The one without ``Bosh_ip`` is the director.
 
+Please notice that when the VM hostname length is exactly 64, the deployment is failing due to ssh problem. This is Softlayer’s limitation which can’t be fixed in a short term. We have a work around in the CPI that when the hostname with 64 characters is identified, a padding "-1" is appended to make it longer than 64.
```
Comment (maximilien) [inline softlayer-cpi.html.md.erb]: @gu-bin please add link to issue or SL limitation. Thx

Comment (gu-bin) [inline softlayer-cpi.html.md.erb]: The SL ticket can only been read by the same account which opened it. I think it doesn't make sense to paste SL ticket link here as others can't read it.

File: init-softlayer.html.md.erb (hunk 1 above)
Comment (gu-bin) [inline init-softlayer.html.md.erb]: link added to the PR

**Comment by maximilien** [comment]
@cppforlife can we get this in the docs please. Thx

**Pull request merged**